bip-0321.mediawiki

```diff
@@ -0,0 +1,228 @@
+<pre>
+  BIP: 321
+  Layer: Applications
+  Title: URI Scheme
+  Author: Matt Corallo <bip21@bluematt.me>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0321
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-11-15
+  License: BSD-2-Clause
+</pre>
+
+==Copyright==
```

Comment:
If you touch up this BIP, could you perhaps settle on formatting for the header lines? Sometimes they have spaces before and after the equal signs, sometimes there is an empty line, it’s a bit topsy-turvy.